src/index.css

```diff
@@ -14,26 +14,57 @@
   -webkit-text-size-adjust: 100%;
 }
 
-a {
-  font-weight: 500;
-  color: #646cff;
-  text-decoration: inherit;
-}
-a:hover {
-  color: #535bf2;
-}
-
 body {
   margin: 0;
   display: flex;
-  place-items: center;
-  min-width: 320px;
+  align-items: center;
+  justify-content: center;
+  min-width: 500px;
   min-height: 100vh;
+  background-color: #eecbc7; 
+  background-image: url("public/iridescent.jpeg");
+  color: #9c8399; 
+}
+
+.container {
+  display: flex;
+  flex-direction: column; 
+  justify-content: center;
+  align-items: center;
+  background-color: #f6dfd7; 
+  border-radius: 10px;  
+  width: 1150px; 
+  height: 300px;
 }
 
-h1 {
-  font-size: 3.2em;
```

Comment:
![image](https://github.com/bpb2008/MagicQuotes/assets/152167/95102137-f223-4adc-8a94-f10e53a6642d)
Another issue: As this background image is just some vague pattern background where details of the image are not important, perhaps you could instead stretch the image to fill the entire screen instead have it repeat as rows and columns like this?